test_scripts/RC/OnRCStatus/001_notification_by_apps_registration.lua

```diff
@@ -9,7 +9,7 @@
 -- In case:
 -- 1) RC app1 and RC app2 are registered
 -- SDL must:
--- 1) Send OnRCStatus notifications to all registered mobile applications and the HMI
+-- 1) Send OnRCStatus notifications to all registered mobile applications and the HMI with allowed = true
```

Comment:
a RC app register event shall trigger an OnRCStatus notification sent to the newly registered RC app only, not to existing RC apps or non-RC apps, as there is no change to the array of freeModules and allocatedModules